### ISSUE TITLE
Mulriplayer - mod for single platform enables disabled particles

### ISSUE THREAD

**Issue by AByakugans** [kinsi55/BeatSaber_Tweaks55#13]
Hello,
when using multiplayer mods and when I check single-platform usage (to imitate single platform and environment to use Chroma etc) during multiplayer sessions - world particles and maybe more when disabled are enabled here.

**Comment by whymoo** [comment]
The devs for multiplayer extensions refuse to fix this bug on their end and say that it needs to be fixed by each particle-removing mod

**Comment by kinsi55** [comment]
Yeah it certainly is down to the mod removing them to fix it, I just dont have too much time currently for BS modding, sorry about that.

**Comment by whymoo** [comment]
Thanks for letting us know
